actionpack/lib/action_dispatch/middleware/host_authorization.rb

```diff
@@ -92,11 +92,16 @@ def call(env)
         response(format, response_body(request))
       end
 
+      attr_writer :blocked_hosts
+      def blocked_hosts
+        @blocked_hosts || []
+      end
+
       private
         def response_body(request)
           return "" unless request.get_header("action_dispatch.show_detailed_exceptions")
 
-          template = DebugView.new(host: request.host)
+          template = DebugView.new(hosts: blocked_hosts)
```

Comment:
I think this should call the `blocked_hosts` method?
```suggestion
          template = DebugView.new(hosts: blocked_hosts(request))
```